sympy/core/numbers.py

```diff
@@ -975,6 +977,8 @@ def __new__(cls, num, dps=None, prec=None, precision=None):
 
         if dps is None and precision is None:
             dps = 15
+            if type(num).__module__ == 'numpy':
+                return convert_numpy_to_sympy(num)
```

Comment:
Perhaps this is dead code?  Can it be a numpy type when we get here?

Reply:
Totally agree - removed.